### PR TITLE
DOC-2552

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ * ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)* https://harness.atlassian.net/browse/DOC-2552

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ * ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by michaelcretzman — DISMISSED** [review]

File: release-notes/security-testing-orchestration.md [hunk 1]
```diff
@@ -13,6 +13,124 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
+ 			
+## January 29, 2023
```
Comment (KimberlyFields) [inline release-notes/security-testing-orchestration.md:17]: You have to update the "date:" on line 4 to match this date. That date is used to update the RSS feed for the release notes.

File: release-notes/security-testing-orchestration.md [hunk 2]
```diff
@@ -13,6 +13,124 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+### New features
+
+* Checkmarx scans now support a `tool_args` field. You can use this field to run the Checkmarx plugin with specific command-line arguments. To run an incremental scan, for example, `specify tool_args` = `-incremental`. (STO-5041)	
+
+* STO now supports orchestrated scans using [Grype](/docs/security-testing-orchestration/sto-techref-category/grype-scanner-reference). (STO-5161)	
+
+### Fixed issues
+
+N/A
```
Comment (KimberlyFields) [inline release-notes/security-testing-orchestration.md:56]: Replace N/A with "This release does not include fixed issues."

File: release-notes/security-testing-orchestration.md [hunk 3]
```diff
@@ -13,6 +13,124 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+## January 1, 2023	
+
+### New features
+
+* The Issues Details pane has been revised to make it easier to navigate. Raw JSON data now appears at the bottom of each occurrence and is collapsed by default. (STO-4839)	
+
+### Fixed issues
+
+N/A
```
Comment (KimberlyFields) [inline release-notes/security-testing-orchestration.md:66]: Replace N/A with "This release does not include fixed issues."

File: release-notes/security-testing-orchestration.md [hunk 4]
```diff
@@ -13,6 +13,124 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+### New features
+	
+* STO is now supported on Linux (amd64 and arm64 platforms) in Harness Cloud. (STO-5314)	
+
+* Instead of using Service Dependencies for Docker-in-Docker configuration, users should use new Background steps. (STO-5268)
+
+### Fixed issues
+
+N/A
```
Comment (KimberlyFields) [inline release-notes/security-testing-orchestration.md:92]: Replace N/A with "This release does not include fixed issues."

File: release-notes/security-testing-orchestration.md [hunk 5]
```diff
@@ -13,6 +13,124 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+This release does not include new features.	
+
+### Fixed issues
+
+* Fixed an issue where the Security Tests tab on the Pipeline Execution page was periodically reloading in a visually jarring way and losing scroll position. (STO-5208)
+
+* Fixed an issue where the Security Tests tab would show previously-found issues while the scan was in progress. Now this tab does not show results until the scan is complete. (STO-4985)	
+
+* Improved the Security Tests UI to highlight new issues found in the current target only vs. issues also found in the baseline or (if no baseline was specified) the previous scan. (STO-5198)
```
Comment (KimberlyFields) [inline release-notes/security-testing-orchestration.md:119]:
```suggestion
* Improved the Security Tests UI to highlight new issues found in the current target only vs. issues also found in the baseline or in the previous scan if no baseline was specified. (STO-5198)	
```

File: release-notes/security-testing-orchestration.md [hunk 6]
```diff
@@ -13,6 +13,124 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
+ 			
+## January 29, 2023	
+
+### New features
+
+* You can now ingest results from a specific Checkmarx scan. This option is useful for ensuring that a pipeline ingests the scan triggered by a specific event. Add the setting `product_scan_id` = `MY_SCAN_ID` to the Checkmarx step. This overrides the default behavior, which is to ingest results for the most recent scan. (STO-5424)
```
Comment (KimberlyFields) [inline release-notes/security-testing-orchestration.md:21]: Add links to the related docs for all new features.

**Review by KimberlyFields — COMMENTED** [review]
A few comments and suggestions.

**Pull request merged**